keps/sig-node/4603-tune-crashloopbackoff/README.md

```diff
@@ -846,33 +1099,47 @@ https://storage.googleapis.com/k8s-triage/index.html
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
 -->
 
+- Crashlooping container that restarts some number of times (ex 10 times),
+  timestamp the logs and read it back in the test, and expect the diff in those
+  time stamps to be minimum the backoff, with a healthy timeout
 - k8s.io/kubernetes/test/e2e/node/kubelet_perf: for a given percentage of
 heterogenity between "Succeeded" terminating pods, and crashing pods whose
-`restartPolicy: Always``, 
- * what is the load and rate of Pod restart related API requests to the API
-   server?
- * what are the performance (memory, CPU, and pod start latency) effects on the
-   kubelet component? Considering the effects of different plugins (e.g. CSI,
-   CNI)
+`restartPolicy: Always` or `restartPolicy: OnFailure`, 
+  - what is the load and rate of Pod restart related API requests to the API
+    server?
+  - what are the performance (memory, CPU, and pod start latency) effects on the
+    kubelet component? 
+  - With different plugins (e.g. CSI, CNI)
 
 ### Graduation Criteria
 
 #### Alpha
 
-- Changes to existing backoff curve implemented behind a feature flag 
-    1. Front-loaded decay curve for all workloads with `restartPolicy: Always`
-       <<[UNRESOLVED]>>
-    2. Front-loaded, low max cap backoff curve for nodes workloads with XYZ
-       config
-- New XYZ kubelet config convertable to ABC on downgrade or without feature flag
-<<[/UNRESOLVED]>>
-- Metrics implemented to expose pod and container restart policy statistics,
-  exit states, and runtimes
-- Initial e2e tests completed and enabled
-  * Feature flag on or off
-  * <<[UNRESOLVED]>>node upgrade and downgrade path <<[/UNRESOLVED]>>
-- Fix https://github.com/kubernetes/kubernetes/issues/123602 if this blocks the
-  implementation, otherwise beta criteria
+- New `int32 crashloopbackoff.max` field in `KubeletConfiguration` API, validated
```

Comment:
Do you propose the actual API / fieldname anywhere?

Reply:
Ah ya only just hidden here. Added explicitly in [1a5f314](https://github.com/kubernetes/enhancements/pull/4893/commits/1a5f314ad4994a229bd7cf98e1ce9403928397fb)